modules/xmpp/TraceablePeerConnection.js

```diff
@@ -317,7 +318,29 @@ TraceablePeerConnection.prototype.addStream = function (stream, ssrcInfo) {
         this.peerconnection.addStream(stream);
     if (ssrcInfo && ssrcInfo.type === "addMuted") {
         this.sdpConsistency.setPrimarySsrc(ssrcInfo.ssrc.ssrcs[0]);
-        this.simulcast.setSsrcCache(ssrcInfo.ssrc.ssrcs);
+        let simGroup = 
```

Comment:
oh, well actually this is basically doing that by checking for the presence of the sim and fid groups